middleware/apimiddleware.go

```diff
@@ -479,6 +479,24 @@ func AddressPostCtx(next http.Handler) http.Handler {
 	})
 }
 
+// DenySpecificAddr https://github.com/decred/dcrdata/issues/358
+func DenySpecificAddr(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		address := GetAddressCtx(r)
+		if address == "DsQxuVRvS4eaJ42dhQEsCXauMWjvopWgrVg" {
```

Comment:
See if this address is somewhere in dcrd or dcrwallet code.  I'd prefer not to hard code this

Reply:
@chappjc  there is no this address. is there are another way to find this address?